content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md

```diff
@@ -6,7 +6,27 @@ weight: 100
 toc: true
 ---
 
-Creating a vegetation layer is the first and most basic step in creating your dynamic vegetation. The following procedure uses a simple workflow and assets from the Starter Game project.
+Creating a vegetation layer is the most basic step in creating your dynamic vegetation, however a few requirements must be met in order to spawn vegetation successfully.
+
+
+**Pre-requisites/Setup**
+
+* Vegetation assets spawned via a Vegetation Layer Spawner component require a surface to plant on. This can be done one of several ways:
+   * In the following workflow, we established a surface by adding a **Mesh Surface Tag Emitter** component to the *Ground* entity that exists as a part of the *Atom Default Environment* entity hierarchy that is added to each new level.
+   * You can also establish a surface by:
+     - Adding a **Mesh Surface Tag Emitter** component to any other entity with a Mesh component
+     - Adding a **Shape Surface Tag Emitter** component to an entity with a Shape component
+     - Adding [Terrain](/docs/user-guide/components/reference/terrain/layer_spawner) to the level
```

Comment:
Let's update this to use the active voice. Using the past tense to refer to a future step is a potential source of confusion. Also, make sure to end bulleted list items with a period.
```suggestion
To spawn vegetation assets using a Vegetation Layer Spawner component, you must have a surface to plant on. You can establish a surface by doing any of the following:

* Add a **Mesh Surface Tag Emitter** component to the *Ground* entity that exists as a part of the *Atom Default Environment* entity hierarchy that is included with each new level. (This technique is used in the example that follows.)
* Add a **Mesh Surface Tag Emitter** component to any other entity that has a Mesh component.
* Add a **Shape Surface Tag Emitter** component to an entity that has a Shape component.
* Add [Terrain](/docs/user-guide/components/reference/terrain/layer_spawner) to the level.
```

Reply:
Updated with suggested change.